.github/workflows/ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
     - name: Run tests
       working-directory: test
-      run: shellspec -f d
+      run: FAROS_DRY_RUN=1 shellspec -f d
```

Comment:
please also update the README - https://github.com/faros-ai/faros-events-cli#running-the-tests

Reply:
Thanks!!!